I'm sorry, but an uncaught exception occurred.

While running game code:
IOError: Couldn't find file 'bgm/t1.ogg'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/common/_layout/screen_main_menu.rpym", line 28, in script
    python hide:
  File "/home/marquiskurt/Desktop/renpy-6.99.14.3-sdk/renpy/ast.py", line 862, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/home/marquiskurt/Desktop/renpy-6.99.14.3-sdk/renpy/python.py", line 1888, in py_exec_bytecode
    exec bytecode in globals, locals
  File "renpy/common/_layout/screen_main_menu.rpym", line 28, in <module>
    python hide:
  File "renpy/common/_layout/screen_main_menu.rpym", line 35, in _execute_python_hide
    ui.interact()
  File "/home/marquiskurt/Desktop/renpy-6.99.14.3-sdk/renpy/ui.py", line 287, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/home/marquiskurt/Desktop/renpy-6.99.14.3-sdk/renpy/display/core.py", line 2635, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "/home/marquiskurt/Desktop/renpy-6.99.14.3-sdk/renpy/display/core.py", line 3304, in interact_core
    renpy.audio.audio.periodic()
  File "/home/marquiskurt/Desktop/renpy-6.99.14.3-sdk/renpy/audio/audio.py", line 976, in periodic_thread_main
    periodic_pass()
  File "/home/marquiskurt/Desktop/renpy-6.99.14.3-sdk/renpy/audio/audio.py", line 920, in periodic_pass
    c.periodic()
  File "/home/marquiskurt/Desktop/renpy-6.99.14.3-sdk/renpy/audio/audio.py", line 409, in periodic
    topf = load(self.file_prefix + filename + self.file_suffix)
  File "/home/marquiskurt/Desktop/renpy-6.99.14.3-sdk/renpy/audio/audio.py", line 69, in load
    rv = renpy.loader.load(fn)
  File "/home/marquiskurt/Desktop/renpy-6.99.14.3-sdk/renpy/loader.py", line 555, in load
    raise IOError("Couldn't find file '%s'." % name)
IOError: Couldn't find file 'bgm/t1.ogg'.

Linux-4.15.0-20-generic-x86_64-with-debian-buster-sid
Ren'Py 6.99.14.3.3347
Doki Doki: The Angel Returns 0.1.0
Mon May 21 16:11:24 2018
